data/common.yaml

```diff
@@ -20,6 +23,13 @@ mcollective::server_config:
   plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
   registerinterval: 10
   registration: "Choria"
+  logfile: "/var/log/puppetlabs/mcollective.log"
```

Comment:
(I wanted to default federation logging config to the server logfile now that we're setting it - but `$mcollective::server_config["logfile"]` is bad there's no requirement this is set)